third_party/txt/src/txt/paragraph_txt.cc

```diff
@@ -807,6 +807,14 @@ void ParagraphTxt::Layout(double width) {
 
       std::shared_ptr<minikin::FontCollection> minikin_font_collection =
           GetMinikinFontCollectionForStyle(run.style());
+      if (minikin_font_collection == nullptr) {
```

Comment:
Remove these logs.  The logging of the errors in `minikin::FontCollection::init` is sufficient.

Reply:
done